libs/sdk-backend-tiger/src/convertors/fromBackend/MetadataConverter.ts

```diff
@@ -60,10 +61,24 @@ function createLabelMap(
     return keyBy(labels, (t) => t.id);
 }
 
+export function createDataSetMap(
+    included: JsonApiAttributeOutDocument["included"] | undefined,
+): Record<string, JsonApiDatasetOutWithLinks> {
+    if (!included) {
+        return {};
+    }
+
+    const labels = included.filter((include): include is JsonApiDatasetOutWithLinks => {
```

Comment:
please check if you could use the existing typeguard https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/api-client-tiger/src/gd-tiger-model/typeGuards.ts#L105-L107 here